utils.js

```diff
@@ -18,3 +18,22 @@ export const truncate = (desc, someHeight) => {
   }
   return desc;
 };
+
+import React from 'react';
+export const filterHelper = prodArr => {
```

Comment:
Is this to return the categories? i'm not sure.

product -> prodArr.map -> array of cat names for each product -> if the array index of the element is the current index, which would always be true? and then return the elem. But filter needs to return either true or false and returns a list of all the elements that had true returned for them. I think here as long the elem is truthy it would be placed in the array returned from filter. I'd fix up the filter right now but I'm not sure exactly what you want filtered out.